pages/linux/synoupgrade.md

```diff
@@ -0,0 +1,36 @@
+# synoupgrade
+
+> Upgrade a Synology DiskStation Manager (DSM) from the command-line.
+> More information: <https://www.synology.com/dsm>.
+
+- Check upgrades printing the result:
+
+`sudo synoupgrade --check`
+
+- Check for patches without upgrading the DSM version:
+
+`sudo synoupgrade --check-smallupdate`
+
+- Download latest upgrade available (use `--download-smallupdate` for patches):
+
+`sudo synoupgrade --download`
+
+- Start the upgrade process:
+
+`sudo synoupgrade --start`
+
+- Upgrade to the latest version automatically:
```

Comment:
So, this is like `--download --start`? Should we maybe include that in the description?

Reply:
Yeah, I think is a combination of `--download` and `--start`.